### ISSUE TITLE
Don't flag window.location.href.indexOf()

### ISSUE THREAD

**Issue by GaryJones** [Automattic/VIP-Coding-Standards#393]
## Bug Description

We currently check for `window.location.href`, but in the case of it being followed by something like `.indexOf('?')`, it doesn't make sense to flag it.

## Minimal Code Snippet

```js
var foo = window.location.href; // Flag this, along with other existing window. items.
if ( window.location.href.indexOf('?') ) { // Don't flag this.
```

- [X] I have verified the issue still exists in the `master` branch of VIPCS.

**Comment by GaryJones** [comment]
As the JavaScript sniffs have been deprecated for VIPCS 3.1.0, I'm closing this out.
